druid/examples/svg.rs

```diff
@@ -0,0 +1,183 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This example shows how to draw an SVG.
+//!
+//! Requires the non-default "svg" feature to be enabled:
+//! `cargo run --example svg --features "svg"`
+
+use druid::widget::{Flex, Svg, WidgetExt};
+use druid::{AppLauncher, Widget, WindowDesc};
+
+const TIGER: &str = r###"
```

Comment:
I think this would be better off as an `include_str`.

Reply:
oops yes I missed this file, that's a good suggestion.